src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -131,6 +131,8 @@ typedef struct client_channel_channel_data {
   grpc_core::UniquePtr<char> info_service_config_json;
   /* backpointer to grpc_channel's channelz node */
   grpc_core::channelz::ClientChannelNode* channelz_channel;
+  /* caches if the last resolution event led to zero addresses */
+  bool previous_resolution_zero_num_addresses;
```

Comment:
Suggest reversing the semantics of this: it should be true if the previous resolution had a non-zero number of addresses.  I think that makes it a little more intuitive to understand.

Reply:
Done